bin/xsd2md.go

```diff
@@ -0,0 +1,175 @@
+package main
```

Comment:
This takes the XSD files given by the product owner and transforms it to easier to read Markdown as well as templates what the components may look like.

Couldn't check in the sources or artifacts due to sensitivity.